sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 ## Release History
 
-### 3.0.0-beta.2 (Unreleased)
-
+## 3.0.0-beta.2 (2020-09-17)
```

Comment:
@weshaggard Could you confirm if this is the correct versioning strategy if the package has updated its group and artifact id.

![image](https://user-images.githubusercontent.com/16845631/93538154-415baf80-f902-11ea-841c-c9806f71e88b.png)

Reply:
@samvaity @weshaggard - group id is same -> `com.azure`, this is just an artifact id upgrade - 
from `azure-spring-data-2-3-cosmos` to `azure-spring-data-cosmos`

Reply:
We could reset the version but I think just incrementing it here is probably ok assuming we still plan to release this as a 3.0.0 GA.

Reply:
@weshaggard - yes that is the plan - this is supposed to go GA as 3.0.0